src/test/java/io/github/hildi/can/service/StandardJavaSerializationServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package io.github.hildi.can.service;
+
+import io.github.hildi.can.exceptions.DeserializeException;
+import io.github.hildi.can.exceptions.FileIsNotExistsException;
+import io.github.hildi.can.exceptions.NotWritableFileException;
+import io.github.hildi.can.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+
+/**
+ * Created by Serhii Hildi on 15.03.19.
+ */
+class StandardJavaSerializationServiceTest {
+
+    private File file;
+    private StandardJavaSerializationService service;
+    private User user;
+    private File mock;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        service = new StandardJavaSerializationService();
+        file = File.createTempFile("temp","txt");
+        mock = mock(File.class);
+        user = new User(1L, "Lord");
+    }
+
+    @Test
+    void shouldReturnTrueIfSerializeDataEqualsDeserializeData() {
```

Comment:
too many details in the name.
it should describe a business idea, for example: `serializedAndThenDeserializedInstancestMustBeEqual`